client/src/auth/auth.js

```diff
@@ -34,7 +35,16 @@ const auth = {
   },
 
   isAuthenticated() {
-    return localStorage.getItem('jwt');
+    const token = jwt.decode(localStorage.getItem('jwt'));
+    if (token) {
+      const { exp: expInSecs } = token;
```

Comment:
This looks a bit complex.

Server can send timestamp, client can generate current timestamp and compare both.

`Date.now() < timestamp`

Reply:
If it is a convention to send seconds as expiry. Then be it.

Reply:
Token generated by "jsonwebtoken" package gives expiry in seconds.